sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TraceHelper.cs

```diff
@@ -48,7 +48,9 @@ internal static List<TelemetryItem> OtelToAzureMonitorTrace(Batch<Activity> batc
                             telemetryItem.Data = new MonitorBase
                             {
                                 BaseType = "RequestData",
-                                BaseData = new RequestData(Version, activity, ref activityTagsProcessor),
+                                BaseData = activityTagsProcessor.activityType.HasFlag(OperationType.V2)
+                                                ? new RequestData(Version, activity, ref activityTagsProcessor, schemaVersion: "V2")
+                                                : new RequestData(Version, activity, ref activityTagsProcessor)
                             };
```

Comment:
It looks like there are very few changes between `RequestData` and `RequestDataV2`.
Would you consider merging these two constructors?
If we did that, we wouldn't need the condition here.